.circleci/config.yml

```diff
@@ -33,30 +33,17 @@ jobs:
       - run: bash <(curl -s https://codecov.io/bash)
       - store_test_results:
           path: build/reports
-  facebook_utils:
+  utils:
     <<: *defaults
     steps:
       - checkout
       - run: *prepare
       - run: bundle exec rake test:facebook_utils:ios
-      - run: bash <(curl -s https://codecov.io/bash)
-      - store_test_results:
-          path: build/reports
-  twitter_utils:
-    <<: *defaults
-    steps:
-      - checkout
-      - run: *prepare
       - run: bundle exec rake test:twitter_utils:ios
+      - run: bundle exec rake test:parseui:all
       - run: bash <(curl -s https://codecov.io/bash)
       - store_test_results:
           path: build/reports
-  parseui:
-    <<: *defaults
-    steps:
-      - checkout
-      - run: *prepare
-      - run: bundle exec rake test:parseui:all
```

Comment:
I'm not sure whether it is helpful to combine the parseui, Facebook and twitter tests. Won't this just make it harder to debug because it will take an extra step to find out what 'part' is failing?

Reply:
@TomWFox Thank you for the review.

I think it doesn't affect debug. It just builds the others on ci steps. 

I believe that We can easily find what's failing in circle/ci logs.

There is no failing on the local machine. And It sometimes passes all tests on ci/circle: utils.

Reply:
I suppose I was just thinking that in the GitHub interface it just makes it a bit easier, not major but it just seems a bit better separated. 

Was there a particular reason you combined them?

Reply:
@TomWFox There was no reason. This PR is based flovilmart PR #1388, and I changed Xcode, ruby and build settings. But I agree with you. Hmm, Would you like to change circle ci this PR?